Enterprise-CMCS/macpro-mdct-mcr
# Small css updates

### Description

A couple of small css updates.
1. Fixed text wrap issue for plan text on export page, increased line height to match general line styling.
2. Focus on input textbox needed an increase in margin to increase distance between focus outline and text above it.

### Related ticket(s)


MDCT-2341
MDCT-2100

---
### How to test

Test long plan name wrap issue
1. Select a MCPAR program
2. Go to Add Plans page
3. Enter a few plans with an obscenely long name (i.e. "the name of a really long plan to test text wrapping on the print pdf")
4. On the Export page, scroll down to (Section D: Plan-Level Indicators), there should be adequate spacing on the title

Test padding for focus styling
1. Select a MCPAR program
2. Any input that is under a paragraph of text should have adequate spacing for the focus box

### Important updates


N/A

---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
